runner/runner.go

```diff
@@ -2267,6 +2267,9 @@ func (r *Runner) skipCDNPort(host string, port string) bool {
 		return false
 	}
 
+	if sliceutil.Contains(r.options.Exclude, "cdn") && isCdnIP {
```

Comment:
we can use `r.scanopts.ExcludeCDN` instead of `sliceutil.Contains(r.options.Exclude, "cdn")`. But I think there is no need for that since we're checking at the beginning of this func.

Reply:
> Also, we're changing the behavior of this function by only checking against `isCdnIP`. Intented?

Previously observed that the logic isn't working without custom ports. But this seems conflicting, Updated the logic to check on custom ports with existing behaviour. can you test now?
cc: @dogancanbakir